dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report.repo.ts

```diff
@@ -236,4 +236,24 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
       },
     });
   },
+
+  /**
+   * Finds the years where the bank with the supplied id submitted
+   * a utilisation report
+   * @param bankId - The bank id
+   * @returns A set of years where the bank submitted a report
+   */
+  async findReportingYearsByBankId(bankId: string): Promise<Set<number>> {
```

Comment:
Considering that this only gets tested in the api tests, do we think it is worth adding a unit test?